crates/telio-nurse/src/nurse.rs

```diff
@@ -42,6 +42,8 @@ pub struct NurseIo<'a> {
     pub config_update_channel: Option<mc_chan::Tx<Box<MeshConfigUpdateEvent>>>,
     /// Event channel to manual trigger a collection
     pub collection_trigger_channel: Option<mc_chan::Tx<Box<()>>>,
```

Comment:
Also let's switch to (), boxing does not really make sence here,

Reply:
Fixed